env0/provider.go

```diff
@@ -59,6 +59,7 @@ func Provider(version string) plugin.ProviderFunc {
 				"env0_team_project_assignment":              resourceTeamProjectAssignment(),
 				"env0_team":                                 resourceTeam(),
 				"env0_environment":                          resourceEnvironment(),
+				"env0_environment_scheduling":               resourceEnvironmentScheduling(),
```

Comment:
what about data?

Reply:
There is no real usage of having a data resource for environment scheduling. 
What would you use out of it? the cron expression?

Reply:
🤷 
we can leave that behind until someone will ask